pkg/util/util.go

```diff
@@ -78,27 +77,27 @@ func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return roundedUp
 }
 
-// ConvertTagsToMap convert the tags from string to map
+// ConvertTagsToMap convert the tags from string to map, default tagDelimiter is ","
 // the valid tags format is "key1=value1,key2=value2", which could be converted to
 // {"key1": "value1", "key2": "value2"}
-func ConvertTagsToMap(tags string) (map[string]string, error) {
+func ConvertTagsToMap(tags string, tagsDelimiter string) (map[string]string, error) {
```

Comment:
`tagsDelimiter` could be empty, if it's empty, then it should be `,`

Reply:
pls also add unit test for `ConvertTagsToMap`, e.g. `tagsDelimiter` is empty, `,`, `;`

Reply:
done